packages/teleport/src/cluster/components/Cluster.tsx

```diff
@@ -71,14 +71,16 @@ export default function Cluster() {
       <TopBar onClickClusterInfo={handleViewClusterInfo} />
       <VerticalSplit>
         <SideNav />
-        <Switch>
-          <Redirect
-            exact
-            from={cfg.routes.cluster}
-            to={cfg.routes.clusterNodes}
-          />
-          {$features}
-        </Switch>
+        <Flex width="100%" style={{ overflowX: 'auto' }}>
```

Comment:
Lets remove this part as it feels unnecessary. I will take a look at the layout myself and put it back if needed.

Reply:
I reverted the styles back to original for both dashboard and cluster layout